tests/test_join.py

```diff
@@ -0,0 +1,249 @@
+import json
+import pytest
+import re
+from typing import Any, MutableMapping, Optional, Sequence, Tuple
+
+from snuba_sdk.column import Column
+from snuba_sdk.conditions import Condition, Op, Or
+from snuba_sdk.entity import Entity
+from snuba_sdk.function import Function
+from snuba_sdk.join import Join, Relationship
+from snuba_sdk.orderby import Direction, OrderBy
+from snuba_sdk.query import Query
+from snuba_sdk.query_visitors import InvalidQuery
+
+tests = [
+    pytest.param(
+        Query(
+            "discover",
+            Join([Relationship(Entity("events", "e"), "has", Entity("spans", "s"))]),
+        )
```

Comment:
sadly spans are not a thing anymore. Up to you, but you may want to reference groupedmessage instead so people do not get confused.

Reply:
Yeah that's a good point.